### ISSUE TITLE
guake is super-scrolled-down

### ISSUE THREAD

**Issue by aviau** [Guake/guake#784]
![screenshot from 2016-08-03 21-09-06](https://cloud.githubusercontent.com/assets/2706882/17387110/b14d3486-59be-11e6-9953-322cb11794dc.png)
![screenshot from 2016-08-03 21-09-16](https://cloud.githubusercontent.com/assets/2706882/17387111/b252907e-59be-11e6-93c0-d489a91e1772.png)

For some reason when I bring up guake it can be scrolled down **below** the most recent line (see first picture).

It causes the terminal to be black, because its scrolled after anything that was printed. Pressing enter or any character will solve the problem.

Note that I use a tiling window manager.

**Comment by Jehan** [comment]
Hi,

I have this problem too since I updated from Fedora 23 to 24. Every time I show guake terminal, it would appear scrolled down more, until it disappears totally. Hitting any key (even a key which doesn't display anything like an arrow) brings back the displayed text where it should be.

This is actually extremely bothering and makes guake hardly usable (since I show/hide it constantly, which is exactly why I hide the concept of a drop/down terminal which I can switch off/on with a single key).

**Comment by Jehan** [comment]
After tweaking a bit I found a workaround. Far from being perfect, but better than the alternative: I uncheck the "Start fullscreen" option and make the guake window slightly smaller than the actual screen height. This seems to have stopped the annoying rescrolling of the shell content when guake window is exposed.

Basically to trigger the scrolling issue, guake has to be fullscreen or have the screen height.

**Comment by Zykr** [comment]
I have this same issue, but I never use guake fullscreen. I have it set to 50% of the screen height and the full width. It usually happens when I turn off my screen for a while and then switch to another open tab once I turn it back on. I'm using XFCE on Debian Unstable if it matters. As @aviau mentioned, pressing any key restores the scroll position, but this could cause unintentional input to programs running in those tabs.

**Comment by fprawits** [comment]
Hi, I just noticed this thread after posting the following issue myself a couple of days ago:

[https://github.com/Guake/guake/issues/832](url)

I am fairly certain that I am experiencing the same issue as the one you are describing; starting guake in non-fullscreen mode indeed "solves" this issue partially.  Here are my observations:

1. The scrolling issue only starts to appear if guake is either started in fullscreen or if I toggle to fullsceen by hotkey (currently using F11).
2. From what I noticed the amount of scrolling depends on how much of the vertial height guake is allowed to use. If I set it to ~95% of the screen height I scroll only by one line, if I set the height to ~30%, I can get the whole terminal to be blank with showing/hiding guake just two times (F12 toggles) as can be seen on the attached pictures in the issue I opened.

So basically guake fullsceen mode currently seems to have a bug that seems to be a regression, since I have been using guake for a long time and only started noticing it after I did an update.

**Comment by fprawits** [comment]
Just found out that this issue is additionally already discussed [here](https://github.com/Guake/guake/issues/438) and [here](https://github.com/Guake/guake/issues/308l)

**Comment by Jehan** [comment]
A bunch of libs just updated on my Fedora 25 and the problem has gotten much worse.
Here the list of what was altered on my system (in particular a bunch of python libs):

Packages Altered:
    Install bzr-2.7.0-14.fc25.x86_64                 @fedora
    Install flatpak-builder-0.8.1-1.fc25.x86_64      @updates
    Install python-enum34-1.0.4-6.fc25.noarch        @fedora
    Install python-gssapi-1.2.0-2.fc25.x86_64        @fedora
    Install python-idna-2.0-4.fc25.noarch            @fedora
    Install python-ipaddress-1.0.16-3.fc25.noarch    @fedora
    Install python-pycparser-2.14-7.fc25.noarch      @fedora
    Install python-pycurl-7.43.0-4.fc25.x86_64       @fedora
    Install python-six-1.10.0-3.fc25.noarch          @fedora
    Install python2-cffi-1.7.0-2.fc25.x86_64         @fedora
    Install python2-cryptography-1.5.3-3.fc25.x86_64 @updates
    Install python2-decorator-4.0.10-3.fc25.noarch   @fedora
    Install python2-paramiko-2.1.1-2.fc25.noarch     @updates
    Install python2-ply-3.8-2.fc25.noarch            @fedora
    Install python2-pyasn1-0.1.9-7.fc25.1.noarch     @fedora

Anyway now I can't use the full screen width anymore if I don't want the problem to appear. This is very inconvenient.

**Comment by pmaselkowski** [comment]
I've set main window height to maximum, both blank screen and scrolling issues gone, but that's not the same guake 👎 I could as well use maximized term (except nicer shortcut). It seems like height (scroll height?) is not properly computed. Did not notice this issue in 0.8.6. Will try do downgrade, and post the results here as well.

**Comment by langpavel** [comment]
0.8.8 too :disappointed:

**Comment by tejaskale** [comment]
Can confirm, still exists in 0.8.8. 
Workaround is to set main window height to maximum and not use fullscreen.

**Comment by Davidy22** [comment]
A rewrite was done, issue no longer appears to occur and no reports submitted since the rewrite, closing.
